### PR TITLE
Fixes #12141 - Add unique id for container object name on 'All containers' page

### PR DESCRIPTION
**Pull request by oshtaier**

### CODE REVIEW

**Comment by omaciel** [comment]
@daviddavis can you review this for us?

File: app/views/containers/_list.html.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@
   <tbody>
   <% containers.each do |container| %>
     <tr>
-      <td class="ellipsis text-center"><%= link_to_container(container, resource) %></td>
+      <td class="ellipsis text-center" id="<%= container.name[1..container.name.length] %>"><%= link_to_container(container, resource) %></td>
```
Comment (daviddavis) [inline app/views/containers/_list.html.erb]: You can just do `container.name[1..-1]` to be more concise.

**Comment by oshtaier** [comment]
Thanks for your feedback, that seems much more elegant solution :).
I saw that you proposed approach with regexp that is also should be fine, but as you removed that part, so I implemented your first idea.

**Comment by eLobato** [comment]
[test]

**Comment by oshtaier** [comment]
It seems that tests are passed, 1 failure doesn't relate to our fix

**Comment by eLobato** [comment]
Merged as d6bc96e60677b9b629c69af5f82c57698b476816, thanks @oshtaier !

**Pull request closed without merging**